### PR TITLE
Add Crypto Policy for Kerberos

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- The rule ensures that kerberos adheres to the system-wide crypto policy.

#### Rationale:

- Kerberos could be configured to opt-out of the system-wide crypto policies.

### CODE REVIEW

**Comment by redhatrises** [comment]
@yuumasato the content should reside under `linux_os/guide/system/software/integrity/crypto`.

**Comment by yuumasato** [comment]
I have relocated the rule.

**Comment by redhatrises** [comment]
@yuumasato one more thing, please remove the `<br/>` from OCIL.

**Comment by yuumasato** [comment]
@redhatrises fixed, thanks for review.

File: linux_os/guide/system/software/integrity/crypto/configure_kerberos_crypto_policy/ansible/fedora.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_fedora
+# reboot = false
```
Comment (matusmarhefka) [inline linux_os/guide/system/software/integrity/crypto/configure_kerberos_crypto_policy/ansible/fedora.yml]: Shouldn't this be rather `true`? If the configuration is already loaded linking the file won't reload it right?

Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_kerberos_crypto_policy/ansible/fedora.yml]: The service can be either restarted, or the system rebooted.
I have updated the remediation metadata in ansible fix, and added metadata to bash fix.

**Comment by matusmarhefka** [comment]
LGTM

**Pull request merged**